import-export-cli/utils/envManagementUtils.go

```diff
@@ -353,4 +353,15 @@ func GetInternalTokenEndpointOfEnv(env, filePath string ) string {
 		internalTokenEndpoint = GetTokenEndPointFromPublisherEndpoint(publisherEndpointOfEnv)
 	}
 	return internalTokenEndpoint
-}
\ No newline at end of file
+}
+
+//Get token endpoint for Token revocation
```

Comment:
Shall we add another comment explaining the function arguments?

Reply:
Fixed with c672258dbfed4a533e8a19dc8f1ef5050d4bd02e